plotnine/qplot.py

```diff
@@ -15,6 +15,8 @@
 from .utils import Registry, is_string, array_kind
 from .exceptions import PlotnineError, PlotnineWarning
 from .scales import scale_x_log10, scale_y_log10
+from .scales import xlim as scales_xlim
+from .scales import ylim as scales_ylim
```

Comment:
Thoughts on a better solution for this naming conflict?

Reply:
Use the general `lims` instead of the specialised functions.